src/sdl2_compat.h

```diff
@@ -749,6 +749,33 @@ struct SDL_SysWMinfo
 
 typedef struct SDL_SysWMinfo SDL_SysWMinfo;
 
+typedef enum SDL_WindowEventID
```

Comment:
Maybe move this above, just after `#define SDL2_WINDOWEVENT 0x200`? (Your git seems to be a bit outdated.) And _maybe_ rename to SDL2_WindowEventID for consistency??

Reply:
Done!